test/testdata

```diff
@@ -133,7 +133,8 @@ Scan      Jaguar       JaguarScanTest       basicJaguar8.3        dvb_scan.out
 
 SP        ADF         ADFSPTest             basicADF2007.01       dvb_sp_b.adfout
 SP        ADF         ADFSPTest             basicADF2013.01       dvb_sp_b.adfout
-SP        DALTON      DALTONSPTest          basicDALTON-2013      b3lyp_energy_dvb_sp.out
+SP        DALTON      DALTONSPTest          basicDALTON-2013      sp_b3lyp_dvb.out
+SP        DALTON      DALTONSPTest          basicDALTON-2013      sp_rhf_dvb.out
```

Comment:
You actually don't need two files; you can have just a .dal file, with the contents of the .mol file at the top, a blank line, and then the job directives.

Which raises the question: should geometries be combined with the inputs, or kept separate where possible? Not all programs allow the separation.